### PR TITLE
Update README.md and remove references to Bazel

### PR DESCRIPTION
**Pull request by brendanhay**
Removes the references to Bazel and adds instructions for using Amazonka as a source/git dependency.

There's still no analogue to Bazel's `//:docs` target to build the website documentation - and as `standalone-haddock` is dead at this point it looks like waiting for [cabal haddock-project](https://github.com/haskell/cabal/pull/8162) to become generally available is more sane than the usual patching/templating/modifying Haddock's output to support a standalone bundle suitable and custom index/landing page for the website. 

See:

* https://coot.me/posts/cabal-haddock-project.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,29 +87,53 @@ nix-env -iA cachix -f https://cachix.org/api/v1/install
...
+
+If the Nix installation is in multi-user mode, don’t forget to restart the nix-daemon. 
+
 ### 3. Enter a Nix Shell
 
 The build tools are installed and activated upon entering a [Nix] shell, which is achieved by running the following command in the root of the repository:
 
 ```bash
-nix-shell
+nix shell
```
Comment (endgame) [inline README.md:105]:
```suggestion
nix develop
```

File: README.md [hunk 2]
```diff
@@ -92,29 +87,53 @@ nix-env -iA cachix -f https://cachix.org/api/v1/install
...
+└───<system>
+    ├───default: development environment 'amazonka-ghc944'
+    ├───ghc810:  development environment 'amazonka-ghc8107'
+    ├───ghc90:   development environment 'amazonka-ghc902'
+    ├───ghc92:   development environment 'amazonka-ghc927'
+    ├───ghc94:   development environment 'amazonka-ghc944'
+    └───ghc96:   development environment 'amazonka-ghc961'
+```
+
+And then running `nix shell` for the desired version from the attribute list above:
```
Comment (endgame) [inline README.md:122]:
```suggestion
And then running `nix develop` for the desired version from the attribute list above:
```

File: README.md [hunk 3]
```diff
@@ -92,29 +87,53 @@ nix-env -iA cachix -f https://cachix.org/api/v1/install
...
+
+``` bash
+nix shell '.#ghc90'
```
Comment (endgame) [inline README.md:125]:
```suggestion
nix develop '.#ghc90'
```

File: README.md [hunk 4]
```diff
@@ -126,75 +145,91 @@ Or the entire project (which will take a very long time!):
...
 
-## Running the Code Generator
+## Amazonka as a Git Dependency
 
-The [gen](gen) Bazel package contains code generators for synthesising Haskell data types, packages, and configuration from the botocore service definitions.
+If there are as-yet-unreleased features or fixes that have yet to make it to Hackage, you can use the `main` (or another) development branch by declaring Amazonka as a Git dependency by following the Cabal or Stack instructions below.
 
-[scripts/generate](scripts/generate) will run the code generator for all services configured in [config/services](config/services), for example:
+> Note: `amazonka-core` is a required dependency of the main `amazonka` package, in addition to `amazonka-sts` and `amazonka-sso` due to `sts:AssumeRoleWithWebIdentity` and SSO authentication and authorization support. These required dependencies can then be supplemented by any additional service libraries you use from `lib/services/amazonka-<service>`.
```
Comment (endgame) [inline README.md:186]: AWS SSO still has that internal name, but maybe we say "SSO via AWS IAM Identity Center"?

**Review by endgame — APPROVED** [review]
Some very minor nits, but once they're done you should merge this. Thank you.

Is this it? Are we go for 2.0 RC 2?

**Comment by brendanhay** [comment]
> Is this it? Are we go for 2.0 RC 2?

Yes - although if there's no clear objective for RC2 I'd also be happy to just go for a 2.0 release proper?

**Comment by endgame** [comment]
I'd like to get one more round of botocore updates and checks for new regions etc in before we go to Hackage.

**Pull request merged**